src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -122,15 +122,17 @@
                                         :accessibility-label :help-button
                                         :action-fn #(re-frame/dispatch [:navigate-to :help-center])}]
      [profile.components/settings-item-separator]
+     [profile.components/settings-item {:label-kw :t/about-app
+                                        :accessibility-label :about-button
+                                        :action-fn #(re-frame/dispatch [:navigate-to :about-app])}]
```

Comment:
Please align map values

Reply:
We need to add this to `cljfmt`, otherwise, this will continue happening. Funny thing, it is not properly formatted three lines above as well... :)